src/main/scala/rocket/RocketCore.scala

```diff
@@ -875,8 +875,8 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
   coreMonitorBundle.timer := csr.io.time(31,0)
   coreMonitorBundle.valid := csr.io.trace(0).valid && !csr.io.trace(0).exception
   coreMonitorBundle.pc := csr.io.trace(0).iaddr(vaddrBitsExtended-1, 0).sextTo(xLen)
-  coreMonitorBundle.wrdst := rf_waddr
-  coreMonitorBundle.wren := rf_wen && !(wb_set_sboard && wb_wen)
+  coreMonitorBundle.wrdst := Mux(wb_ctrl.wxd || wb_ctrl.wfd, wb_waddr, 0.U)
+  coreMonitorBundle.wren := wb_wen && !(wb_set_sboard && wb_wen)
```

Comment:
This change removes the ability to see long-latency writebacks in the log.  That's OK, but might be useful to add a second printout for those in the future.

Reply:
yeah that is still a hole: logging long-latency writebacks after commit, potentially out-of-order

Reply:
we're thinking of how to do this outside of rocket-chip, and we'll likely pursue the idea of a second printout for long-latency writebacks there